test/system/src/test/java/io/pravega/test/system/MultiReaderTxnWriterWithFailoverTest.java

```diff
@@ -67,6 +67,7 @@
     private static final int NUM_WRITERS = 5;
     private static final int NUM_READERS = 5;
     private static final int ZK_DEFAULT_SESSION_TIMEOUT = 60000;
```

Comment:
As i understand the default session timeout is 30 seconds and not 60 seconds, right?